packages/web/src/index.css

```diff
@@ -390,23 +394,28 @@
 .cm-editor * {
   font-family: 'IBM Plex Mono', ui-monospace, 'SFMono-Regular', 'Menlo', 'Monaco', 'Consolas',
     'Liberation Mono', 'Courier New', 'monospace';
+}
+
+.cm-editor {
   font-size: 14px;
 }
 
 .cm-tooltip {
   @apply overflow-hidden;
   @apply !bg-background;
   @apply border;
-  @apply rounded-sm;
   @apply shadow-md;
   @apply !border-border;
+  font-size: 14px;
```

Comment:
isn't 14px the same size as the editor? In which case this is not useful. Didn't we want it to be smaller?

Reply:
it gets override by tw anyways but just pulled it since it's empty junk